docs/writing-easyconfig-files.md

```diff
@@ -257,6 +256,62 @@ example:
 checksums = [('sha512', 'f962008105639f58e9a4455c8057933ab0a5e2f43db8340ae1e1afe6dc2d24105bfca3b2e1f79cb242495ca4eb363c9820d8cea6084df9d62c4c3e5211d99266')]
 ```
 
+It is also possible to specify alternative checksums using a tuple of
+checksum elements where any match is sufficient (logical OR).
+This is helpful when the release was updated with changes that don't affect the behavior of the software in any way
+(e.g. only doc changes).
+
+``` python
+checksums = [('mainchecksum', 'alternativechecksum')]  # Placeholders used
```

Comment:
I would use more obvious placeholders here, and use equal length values:

```suggestion
``` python
checksums = [('0123456789...abcdef', 'fedcba...9876543210')]
```